packages/wonder-blocks-link/src/components/link-core.tsx

```diff
@@ -167,6 +173,63 @@ const sharedStyles = StyleSheet.create({
     },
 });
 
+const action = semanticColor.action.outlined.progressive;
+
+// NOTE: This color is only used here.
+const pink = "#fa50ae";
+
+/**
+ * TODO(WB-1862): Move this to a shared theme file.
+ */
+const theme = {
+    color: {
+        // Primary link color
+        default: {
+            rest: {
+                foreground: action.default.foreground,
+            },
+            hover: {
+                foreground: action.hover.foreground,
+            },
+            focus: {
+                border: semanticColor.border.focus,
+                foreground: action.hover.foreground,
+            },
+            press: {
+                foreground: action.press.foreground,
+            },
+            restVisited: {
+                foreground: color.purple,
+            },
+            pressVisited: {
+                foreground: mix(color.offBlack32, color.purple),
```

Comment:
For my own learning, how do we know when to use the color utils like `mix`?

Reply:
TBH this shouldn't be used too much these days as most of the mixed colors are codified as primitive colors. This was originally introduced to mix colors in the past due to the limitation in our previous primitive colors, but now that we include more shades, this shouldn't be needed anymore (mostly).